admin-manual/installation-setup/installation/install-advanced.rst

```diff
@@ -39,7 +39,7 @@ by installing the following services on separate machines:
 * Gearman
 * MySQL
 
-For help, send an email to the `archivematica-tech`_ mailing list.
+For help, ask on the `Archivematica google group`_ for more details.
```

Comment:
Can I suggest I different name for the list? Some examples:
- Archivematica community discussion list
- Archivematica user forum
- Archivematica user mailing list

Reply:
Sounds good you guys. I'll go with *Archivematica user forum*.